### PR TITLE
Removes parasites from raw_milk

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
```SUMMARY: Balance "Removes parasites from raw_milk"```

#### Purpose of change
It makes no sense to get parasites from drinking raw milk.
Raw milk now has a 5% chance to give you foodpoisoning
The duration of the effect is the same as rotting food, meaning that it can last from 6min to 96min 

#### Describe the solution
- [x] Remove parasites field from raw_milk
- [x] Replace it with food poisoning
        Add a `contamination` entry which represent the probability to get foodpoisoning from the comestible.
        `contamination` is not inherited through crafting process
        

#### Testing
Drink raw milk
No effect
Repeat a few times
Get foodpoisoning

#### Additional context
The kind of foodpoisoning you'd get from contaminated milk should last more than 1h30 but I'm re using the foodpoison effect already in the game and the frequency of the vomiting and the stat penalty you get from it would be quite severe if it lasted for a day. It would also be weird if eating rotting food was safer than drinking milk.
However I can make it last longer, or be tied to the `contamination` if it's deemed necessary, we could even go as as far as tying it with the health stat since weak health makes you more susceptible to get foodpoisoning.

Sources:
https://www.cdc.gov/features/rawmilk/index.html
https://en.wikipedia.org/wiki/Raw_milk
https://www.cdc.gov/foodsafety/diseases/staphylococcal.html

Probability: 
from: https://wwwnc.cdc.gov/eid/article/21/1/14-0447_article
> During 2007–2012, a total of 81 outbreaks associated with nonpasteurized milk were reported from 26 states. These outbreaks resulted in 979 illnesses and 73 hospitalizations. No deaths were reported.

from: https://en.wikipedia.org/wiki/Raw_milk
> The Cornell University Food Science Department has compiled data indicating that pathogenic microorganisms are present in between 0.87% and 12.6% of raw milk samples.

So putting contamination probability at 5% sounds reasonnable

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
> It makes no sense to get parasites from drinking raw milk

[CDC thinks different](https://www.cdc.gov/features/rawmilk/index.html).

**Comment by Fris0uman** [comment]
> That’s because it can contain harmful germs, such as Brucella, Campylobacter, Cryptosporidium, E. coli, Listeria, and Salmonella.

Those are not parasites, my plan was to add Staphylococcal Food Poisoning instead of parasites https://www.cdc.gov/foodsafety/diseases/staphylococcal.html

**Comment by Night-Pryanik** [comment]
Consuming raw milk can cause [giardiasis](https://en.wikipedia.org/wiki/Giardiasis), which ain't bacteria, but a parasite.
https://www.gov.mb.ca/health/publichealth/diseases/giardia_lamblia.html

**Comment by Fris0uman** [comment]
> Giardiasis is spread by the fecal-oral route, either directly by person-to-person contact (e.g. diaper changing, sexual practices) or indirectly by eating or drinking fecally contaminated food or water. The infection can also spread by coming in contact with the feces of an infected animal. Fecal material can get into a person’s mouth by contact with the feces of an infected person or animal that is not followed up by proper hand washing.

So I guess it's possible, but that's not inherent to milk, it's just linked to poor hygiene conditions. Which is a thing we don't currently track

**Comment by Night-Pryanik** [comment]
I don't know what else to say. Major healthcare and diseases study centers says in plain text: "Don't drink raw milk, you may get bacteria, viruses, PARASITES", and you don't believe them?

**Comment by Fris0uman** [comment]
The game can currently give you: brain parasites, blood parasites, tape worn and pain cyste from drinking milk. All the links you've provided confirm that you can't get any of those things from drinking raw milk.

What you can get from drinking raw milk is different degree of foodpoisonning ranging from diarrhea to death, and a gut parasite that you could get from any food contaminated with fecal matter.

File: src/consumption.cpp [hunk 1]
```diff
@@ -949,6 +949,14 @@ bool player::eat( item &food, bool force )
         }
     }
 
+    // chance to get foodpoisoning from bacterial contamination
```
Comment (Zireael07) [inline src/consumption.cpp]: *nitpick* space between food and poisoning ;)

File: data/json/items/comestibles/dairy.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "copy-from": "milk",
     "spoils_in": "12 hours",
     "price": 19,
-    "parasites": 11,
+    "contamination": 30,
```
Comment (ymber) [inline data/json/items/comestibles/dairy.json]: 30% is a very high contamination rate. When unpasteurized milk was the norm it was safe enough that a large majority of people didn't get poisoned by it.

**Comment by I-am-Erk** [comment]
@night-pryanik the type of parasitosis they're describing is functionally identical to severe food poisoning. It's called parasitosis because it's neither a virus nor a bacterium.

@fris0uman We should look into making specific types of food poisoning by infectious organism in a later pr. Being able to get giardiasis from unclean water or milk versus campylobacter from rotten food or milk is quite different and tracking them differently would let us put in different treatments

**Comment by Night-Pryanik** [comment]
@I-am-Erk yeah, thanks, now it's clear to me.

**Pull request merged**